### ISSUE TITLE
Implement automatic color/grayscale page detection

### ISSUE THREAD

**Issue by Spladenly** [Spladenly/ai-manga-upscale-colab#3]
Instead of having to manually sort manga pages by color and grayscale, this should be done automatically.

**Comment by Spladenly** [comment]
Turns out, this is not really viable, since most of the manga pages are saved very inconsistently. There is really no reliable and fast way of determining whether an image is truly black-and-white, if it's saved incorrectly (not in grayscale). If manga pages were scanned in rgb, then they have real color data, you can actually see the image changing by a bit, if you discard the color information. In such case, it's really not possible to determine whether it's colored or not.

TLDR; even if this gets implemented, you still have to manually confirm and/or wait some additional time for it to check, which is not really viable on colab
